demo/main.ts

```diff
@@ -1,4 +1,13 @@
-import * as fontoxpath from '../src/index';
+import {
```

Comment:
All of these changes are their own commit that makes the demo page work with the single binary. I'll remove the commit before integration, but it was useful for debugging things